src/components/wrapper/wrapper.jsx

```diff
@@ -186,28 +78,30 @@ export default class Wrapper extends React.Component {
         });
 
         return (
-            <div className="vote-app">
-                {/* TODO: Add cleaner title or no title
-                    <div className="vote-app__title">
-                        Vote
-                    </div>
-                */}
-
-                <div className="vote-app__influence">
-                    <div className="vote-app__top">
-                        <div className="vote-app__influence">
-                            <div className="vote-app__influence-description">
-                                <Influence className="vote-app__influence-section" type="normal"/>
-                                <Influence className="vote-app__influence-section" type="golden"/>
+            <div className={ block }>
+                <div className={ `${block}__influence` }>
+                    <div className={ `${block}__top` }>
+                        <div className={ `${block}__influence` }>
+                            <div className={ `${block}__influence-description` }>
+                                <Influence 
+                                    className={ `${block}__influence-section` } 
+                                    type="normal" />
+                                <Influence 
+                                    className={ `${block}__influence-section`} 
+                                    type="golden" />
                             </div>
 
-                            <div className="vote-app__influence-disclaimer">
+                            <div className={ `${block}__influence-disclaimer` }>
                                 DISCLAIMER: Since this feature is its Alpha stages, the formula for calculating influence may change.
```

Comment:
@pksjce good question, I'm not sure. I think @TheLarkInn or @sokra would know better, I'm assuming we probably we to get some more usage and feedback before removing it, however idk how much people are using it currently.